vendor/ddcloud/resource_vip_pool_member.go

```diff
@@ -45,10 +45,9 @@ func resourceVIPPoolMember() *schema.Resource {
 				Computed: true,
 			},
 			resourceKeyVIPPoolMemberPort: &schema.Schema{
-				Type:     schema.TypeString,
```

Comment:
In the API documentation, it's specified that, if you need the port, then you need the port number if not then, port value should not be specified so that it can add a member with ANY port configuration. 

I have tested it both with and without the port number and it seems to be working